pluralize.js

```diff
@@ -257,9 +262,11 @@
    * @param {string} single
    * @param {string} plural
    */
-  pluralize.addIrregularRule = function (single, plural) {
-    plural = plural.toLowerCase();
-    single = single.toLowerCase();
+  pluralize.addIrregularRule = function (single, plural, { preserveCasing = false } = {}) {
```

Comment:
Unfortunately the rest of this repo is still built to run on very old JS versions and I'd like to keep it that way for now, can you dumb this down to ES5 at a minimum?

Reply:
Happy just to make this a simple boolean flag instead of an options object to make that easier too.

Reply:
Removed the spreading, I figured an options object might be better here to future proof the API incase someone else needs an edge case handled. What do you think?